content/download/verify.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: simplepage
+title: "Verifying Downloads"
+---
+
+== Verifying Jenkins Downloads
+
+NOTE: Jenkins automatically verifies the integrity of Jenkins core updates it downloads from update centers. These instructions apply to manual downloads.
+
+=== WAR File Verification
+
+The Jenkins war file (2.232 and newer, LTS 2.235.3 and newer) are signed by the Jenkins project.
+These signatures can be verified using `jarsigner`, a tool included with the Java runtime.
+Expected output of `jarsigner -verify -verbose jenkins.war`:
+
+[source]
+----
+- Signed by "CN="CDF Binary Project a Series of LF Projects, LLC", OU=Jenkins Project, O="CDF Binary Project a Series of LF Projects, LLC", L=Dover, ST=Delaware, C=US"
+    Digest algorithm: SHA-256
+    Signature algorithm: SHA256withRSA, 4096-bit key
+----
+
+Earlier releases were created and signed by Kohsuke Kawaguchi.
+Expected output of `jarsigner -verify -verbose jenkins.war`:
+
+[source]
+----
+- Signed by "CN=Infradna Inc (Kohsuke Kawaguchi), O=Infradna Inc (Kohsuke Kawaguchi), STREET=4438 Hilton Ave, L=San Jose, ST=California, OID.2.5.4.17=95130, C=US"
+    Digest algorithm: SHA-256
+    Signature algorithm: SHA256withRSA, 2048-bit key
+----
+
+The SHA-256 checksums of the latest weekly and LTS releases are published link:/download[on the downloads page] next to the respective .war download option.
+The SHA-1 and SHA-256 checksums of past releases are published https://updates.jenkins.io/download/war/[here].
+
+
+=== Windows MSI Installers
+
+Windows MSI Installers are signed with the same code signing certificate as the WAR file.
+
+// TODO more details?
+
+=== Linux Package Repositories
+
+Linux package repositories use the following GPG key:
+
+[source]
+----
+pub   rsa4096 2020-03-30 [SC] [expires: 2023-03-30]
+      62A9756BFD780C377CF24BA8FCEF32E745F2C3D5
+uid                      Jenkins Project <jenkinsci-board@googlegroups.com>
+sub   rsa4096 2020-03-30 [E] [expires: 2023-03-30]
+----
+
+// TODO since when, and what was it before?
+
+== Verifying Plugin Downloads
+
+NOTE: Jenkins automatically verifies the integrity of plugins it downloads from update centers. These instructions apply to manual downloads.
+
+To manually download plugin releases, visit the plugin's page on the https://plugins.jenkins.io/[plugin site] and select "Archives".
+That page will list all releases available for download as well as their SHA-1 and SHA-256 checksums.
```

Comment:
Add disclaimer that SHA-1 checksum is not recommended?

Reply:
Anyone who cares enough should know already. But we could link to their Wikipedia pages?

Reply:
We could also remove the definition of checksum algorithms from the sentence and just say "as well as their checksums".